### ISSUE TITLE
AMC and quantization will work for cnn+rnn?

### ISSUE THREAD

**Issue by ryankang95** [NervanaSystems/distiller#457]
hi, I want to know if AMC is useful for neural networks like CNN + LSTM like CRNN or what should i do to apply AMC to CRNN?  
one more that, i tried run quantization for quantizing CRNN, but i'm failed, the parameters of LSTM seem to be lost. so I want to know if the quantization method in distiller work for cnn + rnn networks. 

and if i change code to control layers that will be quantized and compressed in CRNN (such as only execute AMC and quantization for CNN part and ignored RNN part), will it cause accuracy heaily loss heavily?

thank!

**Comment by guyjacob** [comment]
Post-training quantization can be applied to LSTM by replacing the "vanilla" PyTorch `torch.nn.LSTM` module with `distiller.modules.DistillerLSTM`.
To convert all LSTMs in your model to the Distiller variant you can call:
```python
distiller.modules.convert_model_to_distiller_lstm(model)
```
(And see [here](https://github.com/NervanaSystems/distiller/blob/master/examples/word_language_model/quantize_lstm.ipynb) for an example of LSTM post-training quantization)

**Comment by levzlotnik** [comment]
Hi,

Closing this issue as it seems resolved.
